src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter
+            .ofPattern("dd.MM.yyyy");
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate localDateFrom = LocalDate.parse(dateFrom, DATE_FORMAT);
+        LocalDate localDateTo = LocalDate.parse(dateTo, DATE_FORMAT);
+        StringBuilder builder = new StringBuilder();
+
+        builder.append("Report ").append("for ").append("period ")
+                .append(localDateFrom.format(DATE_FORMAT)).append(" - ")
+                .append(localDateTo.format(DATE_FORMAT));
+
+        for (String name : names) {
+            int salary = 0;
+            for (String dates: data) {
```

Comment:
```suggestion
            for (String info : data) {
```
dates is not informative